components/_twig-components/functions/bem.function.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
well, we added it to the install process because we thought we might update it at some point, so... ?